src/columnstore/columnstore_metadata.cpp

```diff
@@ -14,12 +15,57 @@ extern "C" {
 #include "catalog/namespace.h"
 #include "commands/dbcommands.h"
 #include "miscadmin.h"
+#include "utils/array.h"
 #include "utils/builtins.h"
 #include "utils/fmgroids.h"
 #include "utils/lsyscache.h"
 #include "utils/rel.h"
+#include "utils/timestamp.h"
 }
 
+// Put these functions in public anonymous namespace to avoid name conflict within duckdb.
```

Comment:
Unfortunately the function inside of postgres is called by macros, we don't have too many ways to resolve.